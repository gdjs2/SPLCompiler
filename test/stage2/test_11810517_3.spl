int compare1(int x) {
	return 1;
}

int main() {
	int a, b, c = 10;
	
	compare(3);
	return 1;
}