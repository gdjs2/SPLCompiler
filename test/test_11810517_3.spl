float n(){
    m();
    return 1;
}

float m(){
    n();
    return 1;
}