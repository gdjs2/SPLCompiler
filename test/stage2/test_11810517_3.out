Error type 2 at Line 8: function "compare" is not defined.